config-templates/salesforce/users.yaml

```diff
@@ -0,0 +1,24 @@
+data_source:
+  name: salesforce_users
+  owners:
+    - jordan@transformdata.io
```

Comment:
Probably put some filler here unless you want to field questions for companies' questions about their salesforce users :P